CONTRIBUTORS

```diff
@@ -5,31 +5,31 @@ shazbert | https://github.com/shazbert
 dependabot[bot] | https://github.com/apps/dependabot
 gloriousCode | https://github.com/gloriousCode
 dependabot-preview[bot] | https://github.com/apps/dependabot-preview
-xtda | https://github.com/xtda
 gbjk | https://github.com/gbjk
+xtda | https://github.com/xtda
 lrascao | https://github.com/lrascao
-Rots | https://github.com/Rots
-vazha | https://github.com/vazha
 ydm | https://github.com/ydm
+vazha | https://github.com/vazha
+Rots | https://github.com/Rots
 ermalguni | https://github.com/ermalguni
 MadCozBadd | https://github.com/MadCozBadd
 Beadko | https://github.com/Beadko
 vadimzhukck | https://github.com/vadimzhukck
-140am | https://github.com/140am
-marcofranssen | https://github.com/marcofranssen
 geseq | https://github.com/geseq
 samuael | https://github.com/samuael
+marcofranssen | https://github.com/marcofranssen
+140am | https://github.com/140am
 TaltaM | https://github.com/TaltaM
 dackroyd | https://github.com/dackroyd
 cranktakular | https://github.com/cranktakular
 khcchiu | https://github.com/khcchiu
 yangrq1018 | https://github.com/yangrq1018
 woshidama323 | https://github.com/woshidama323
 crackcomm | https://github.com/crackcomm
-azhang | https://github.com/azhang
```

Comment:
Need to reinstate these contributors by hardcoding them into contributors list: https://github.com/thrasher-corp/gocryptotrader/blob/master/cmd/documentation/documentation.go#L175

Reply:
Can you explain this pls?

Reply:
Sure! Azhang, andreygrehov, bretep and ChristianAchilli's contributions have been removed in this PR. The API doesn't return them after a period of time so we hardcode them into this list: https://github.com/thrasher-corp/gocryptotrader/blob/master/cmd/documentation/documentation.go#L175

Reply:
I hope no contributor is missing other than these I added